src/mbed_os_tools/test/tests_spec.py

```diff
@@ -48,13 +49,20 @@ def __init__(self, path, binary_type):
         )
         self.__path = path
         self.__flash_method = binary_type
+        self.__comp_log = compare_log
 
     def get_path(self):
         """
         Gives binary path.
         :return:
         """
         return self.__path
+    def get_comp_log(self):
```

Comment:
I prefer not to abbreviate stuff if the full phrase is still quite short (just a difference of three letters):
```suggestion
    def get_compare_log(self):
```

Reply:
Fixed